src/pages/ModelOfInterest/ArcDiagram/index.tsx

```diff
@@ -202,7 +217,27 @@ const ArcDiagram: React.FC = () => {
   }, []); // Empty dependency array ensures useEffect runs once on component mount
 
   return (
-    <div id="Arc_diagram"></div>
+    <div>
+    <div id="indicator" style={{ position: "absolute", top: "0", right: "0", margin: "20px", marginTop: "70px" }}>
+      <div className="container" style={{ display: "flex", alignItems: "center", marginBottom: "2px", marginTop: "17px" }}>
+        <div className="circle model" style={{ backgroundColor: "currentColor", width: "10px", height: "10px", borderRadius: "50%", marginRight: "5px", marginLeft: "20px"}}></div>
+        <p style={{ fontSize: "13px" }}>Classic (Model)</p>
+      </div>
+
+      <div className="container" style={{ display: "flex", alignItems: "center", marginBottom: "5px", marginTop: "2px" }}>
+        <div className="circle dependency" style={{ backgroundColor: "#EC5800", width: "10px", height: "10px", borderRadius: "50%", marginRight: "5px", marginLeft: "20px"}}></div>
+        <p style={{ fontSize: "13px" }}>LLM (Model)</p>
+      </div>
+
+      <div className="container" style={{ display: "flex", alignItems: "center", marginBottom: "5px", marginTop: "2px" }}>
+        <div className="circle dependency" style={{ backgroundColor: "blue", width: "10px", height: "10px", borderRadius: "50%", marginRight: "5px", marginLeft: "20px"}}></div>
+        <p style={{ fontSize: "13px" }}>Unknown</p>
```

Comment:
@pramit-d We wouldnt be having an Unknown type model. Its either Classic or LLM as of now.

Reply:
@harimohanr updated.